src/components/grid-aware/ArticleSpotlightCard/ArticleSpotlightCard.js

```diff
@@ -5,10 +5,11 @@ import Button from "../../inline/Button";
 
 import s from "./ArticleSpotlightCard.module.css";
 
-const TextCard = ({ eyebrowText, title, description, button }) => (
+const TextCard = ({ eyebrowText, title, subtitle, description, button }) => (
   <div className={s.textCard}>
     <div className={s.eyebrow}>{eyebrowText}</div>
     <div className={s.title}>{title}</div>
+    <div className={s.subtitle}>{subtitle}</div>
```

Comment:
Let's conditionally render this only if `subtitle` is defined.

Reply:
ok